command/notify.go

```diff
@@ -14,6 +14,7 @@ var cmdNotifySet = &Command{
 	Long: `
 Determines if notifications should be used
 `,
+	MaxExpectedArgs: -1,
```

Comment:
Why isn't this 1?

Reply:
`notify` already does its own arguments checking.